rep-2005.rst

```diff
@@ -39,20 +39,53 @@ The list below is curated to include repositories that are:
 
 We aim that all of these repositories are also:
```

Comment:
I agree and I think that we should talk about what it would take to turn the dreams into reality in the next TSC meeting. But I would also urge that we embrace an  iterative improvement model and let this go in as is.

Reply:
During the meeting we discussed a transition from where we are now to requiring these criteria for G Turtle, but there is the catch that the existing list of packages may not meet all criteria by then.